### DISCUSSION TITLE
How to identify photos imported with immich-go from Google Takeout?

### DISCUSSION THREAD

**Discussion by 2600box** [simulot/immich-go#516] [Q&A]
Hello,

I used immich-go to import my google takeout and it went smoothly, and I can albums from the google takeout.

Question: Is there a way to filter for those photos imported with immich-go? 

They are mixed in with my regular and recent photos but I want to be able to go through them as a set.

I cannot see an album just for immich-go or the google takeout, and cannot identify a method to filter...

Thanks in advance for your help!

**Comment by simulot** [comment]
There is currently no way to identify them.
In a next future, uploaded images will have a tag like "immich-go/YYY-MM-DD-HH-MM-SS"

**Reply by TheBestMoshe** [reply]
Can we get this tag added even if the image was skipped because it was already uploaded? I already uploaded my Google Takeout, but I'd like to add this tag to those images.
